internal/databases/sqlserver/log_restore_handler.go

```diff
@@ -56,22 +56,43 @@ func HandleLogRestore(backupName string, untilTS string, dbnames []string, fromn
 	err = runParallel(func(i int) error {
 		dbname := dbnames[i]
 		fromname := fromnames[i]
+		var lastPiece bool
+		var LogsMetadata []BackupProperties
+
 		for _, logBackupName := range logs {
-			ok, err := doesLogBackupContainDB(folder, logBackupName, fromname)
+			baseURL := getLogBackupURL(logBackupName, fromname)
+			basePath := getLogBackupPath(logBackupName, fromname)
+			blobs, err := listBackupBlobs(folder.GetSubFolder(basePath))
```

Comment:
What if blobs is empty list ? eg database `fromname` is not present in particular logBackupName.
Before this patch there was a `doesLogBackupContainDB` check here